design/src/App.js

```diff
@@ -2,19 +2,24 @@ import React, { useState } from 'react';
 import Header from './components/Header/Header.jsx';
 import Cookie from './components/Cookie/Cookie.jsx';
 import './App.scss';
+import Form from './components/Form.jsx';
+import Footer from './components/Footer.jsx';
 
 function App() {
     const [visibleCookie, setVisibleCookie] = useState(true);
 
     return (
         <div className="App">
-            <Header />
+            {/* <Header />
```

Comment:
Maybe remove the code that's commented out all-together, or add a comment as to why it's left behind :)

Reply:
This was done purely for testing reasons, will change it when code goes live :)